config/routes.rb

```diff
@@ -11,5 +11,4 @@
   end
 
   root 'static_pages#home'
```

Comment:
The static_pages_controller.rb is in the controllers folder. It is used to show the initial page. Its view is in the views/static_pages/home.html.erb

Yes, it is working.